dabl-meta.yaml

```diff
@@ -4,16 +4,14 @@
 catalog:
     name: dabl-integration-core
     group_id: com.digitalasset
-    version: 0.5.5
+    version: 0.5.6
     short_description: The Core Pack
     description: Timer and Loopback Integrations
-    release_date: 2020-09-30
```

Comment:
See.... I told you this was deprecated! (Actually, `ddit` is now populating it automatically. :-) )